e2e/snapshot-creators/default.cy.snap.js

```diff
@@ -173,22 +173,11 @@ describe("snapshots", () => {
     // dashboard 1: Orders in a dashboard
     const dashboardDetails = { name: "Orders in a dashboard" };
 
-    cy.createQuestionAndDashboard({ questionDetails, dashboardDetails }).then(
-      ({ body: { id, card_id, dashboard_id } }) => {
```

Comment:
I forgot to update this method call in the [PR](https://github.com/metabase/metabase/pull/29641/commits) that refactored the underlying API endpoint.

It still works with the separate `PUT` request called in the `.then` method, but it's less efficient and the code is cleaner when we just use `cardDetails`.